ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
I assume if the CRC is missing the underlying file can still be used.  CRC missing caused the file to be rebuilt because it was depending on the CRC to work out if it has changed - when Refresh=false, then it's not necessary to execute that check. @dabayliss can you comment please?